app/controllers/support_interface/performance_controller.rb

```diff
@@ -46,17 +46,17 @@ def candidate_survey
     end
 
     def applications_export_for_ucas
-      if FeatureFlag.active?('download_dataset1_from_support_page')
+      if HostingEnvironment.production?
+        # The unauthorized page expects an instance var that's only set in
+        # the dfe_sign_in_controller
```

Comment:
This is a thing because we're overloading the current "unauthorized" page. I think the right thing to do would be name that properly and create a generic "unauthorized" page for generic unauthorized actions?

Reply:
Raising a 404 now, which is better anyway.